src/generator/getAllOptions.ts

```diff
@@ -0,0 +1,10 @@
+import { RulesOptions } from '../rules/rules.options';
+
+export function getAllOptions(): object {
+  return Object.entries(new RulesOptions()).map(o => {
```

Comment:
I think you can do that. But not sure. 

```javascript
return Object.entries(new RulesOptions()).map([name, value] => {
    return {
      name,
      value,
    };
  });
```

Reply:
Argument of type 'any[]' is not assignable to parameter of type '(value: [string, any], index: number, array: [string, any][]) => {}'.
  Type 'any[]' provides no match for the signature '(value: [string, any], index: number, array: [string, any][]): {}'.ts(2345)